api/init.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	"sync"
+
+	"github.com/raintank/metrictank/expr"
+	"gopkg.in/raintank/schema.v1"
+)
+
+const defaultPointSliceSize = 2000
```

Comment:
maxPointsPerReqHard is per request. but a pointslice is a slice of points for a given series.
you're right that it's somewhat arbitrary. you want it low enough to not waste too much memory allocating slices with space that we don't use, and high enough to avoid expensive re-allocation and copying as we need to expand a slice as it gets filled; though the cost of this gets amortized as we can reuse the slice (incl backing array) over time so starting out a little low is not so problematic. 
it's hard to generalize how many points will typically be returned for any given series, across different requests, but I think this is a good enough value that we don't have to worry about for a while.